api/README.md

```diff
@@ -198,7 +198,11 @@ You can deploy the API service via various methods below based off developer pre
   * [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/) via [Skaffold](https://github.com/GoogleContainerTools/skaffold)
   * [Google Cloud App Engine](https://cloud.google.com/appengine/) via [Deployment Manager](https://cloud.google.com/deployment-manager/) and gcloud - TODO
 
-[Deploy Cloud Run](#deploy-cloud-run) on [Anthos](https://cloud.google.com/run/docs/choosing-a-platform#cloud-run-for-anthos) is the _required_ method to enabled fine-grained authorization with [Istio](http://istio.io/). [Managed](https://cloud.google.com/run/docs/choosing-a-platform#cloud-run-fully-managed)is the _preferred_ method to quickly host the DS API Service content and generate a unique URL for consumption.
+[Deploy Cloud Run](#deploy-cloud-run) has two options [Deploy Cloud Run - Managed](#deploy-cloud-run-managed) and [Deploy Cloud Run - Anthos](#deploy-cloud-run-anthos).
+
+* The [Managed](https://cloud.google.com/run/docs/choosing-a-platform#cloud-run-fully-managed) is the _preferred_ technology to deploy, host, and generate a unique URL for consumption. The [API Gateway](https://cloud.google.com/api-gateway) is leverage for fine-grained authorization and security enforcement for the DS API service on Cloud Run Managed.
+
+* [Anthos](https://cloud.google.com/run/docs/choosing-a-platform#cloud-run-for-anthos) is the self-managed Cloud Run deployment on GKE that leverages [Istio](http://istio.io/) for the fine-grained authorization and security enforcement.
```

Comment:
Should we remove this as a suggested option?

Reply:
I think the Cloud Run for Anthos option can be removed or moved to a separate alternative deployment readme. If moved, I add the other deployment options too [deploy-kubernetes](https://github.com/GoogleCloudPlatform/datashare-toolkit/blob/a4c311c1bbefaee6e0aaa3a3349e3aedf887cbd3/api/README.md#deploy-kubernetes) to clean up the main README 
wdyt? 

Thanks,
Chris

Reply:
I think remove it, easier to support one method.